runners/gradle-plugin/src/main/kotlin/org/jetbrains/dokka/gradle/utils.kt

```diff
@@ -17,7 +17,11 @@ internal fun parsePath(path: String): Path = Path.path(path)
 internal val Project.kotlinOrNull: KotlinProjectExtension?
     get() = try {
         project.extensions.findByType()
-    } catch (e: NoClassDefFoundError) {
+    } catch (e: Exception) {
```

Comment:
Debated myself on catching both `NoClassDefFoundError` and `TypeNotPresentException`, but went with `Exception` just in case it changes again.

There might be a better way to do this (without catching exceptions), but I don't think it makes much sense to invest in it now.

Reply:
Note that `NoClassDefFoundError` is not a subtype of `Exception`
Maybe it's better to catch a `Throwable` and then filter it by types, e.g.
```
catch (e: Throwable) {
    when (e) {
         is NoClassDefFoundError, is TypeNotPresentException, is ClassNotFoundException -> null
         else -> throw e
    }
}
```

Reply:
Oh yeah, totally slipped my mind! must've been thinking about `ClassNotFoundException`

fixed